### DISCUSSION TITLE
Menu with (lazyMount: true, unmountOnExit: true) does render on the 0,0 position

### DISCUSSION THREAD

**Discussion by xeinebiu** [chakra-ui/ark#3172] [Q&A]
After updating to the latest version of Ark (4.5.0), all Menu's (sometime on first, sometime after the first trigger) they are showing on the top left on the screen. 

Removing lazyMount:true and unmounOnExit: true does fix the issue but for performance reasons on big list, lazy is required.

https://codesandbox.io/p/devbox/snowy-monad-tl86nz

Cannot reproduce it on the codesandbox. Probably is only related to 0.42.0 on ParkUI.

**Comment by segunadebayo** [comment]
This is now fixed. Please upgrade to the latest version

**Reply by xeinebiu** [reply]
> This is now fixed. Please upgrade to the latest version

After upgrading to the latest version, the client app fails at runtime 

https://github.com/chakra-ui/ark/discussions/3201

**Comment by segunadebayo** [comment]
Closing — menu positioning with `lazyMount` has been fixed.
